doc/mini-howto/Makefile.am

```diff
@@ -2,7 +2,7 @@ dist_pkgdata_DATA = dar-differential-backup-mini-howto.en.html dar-differential-
 
 install-data-hook:
 	$(INSTALL) -d $(DESTDIR)$(pkgdatadir)/mini-howto
-	$(INSTALL) -m 0644 $(dist_pkgdata_DATA) $(DESTDIR)$(pkgdatadir)/mini-howto
+	for f in $(dist_pkgdata_DATA); do $(INSTALL) -m 0644 '$(srcdir)/'"$${f}" $(DESTDIR)$(pkgdatadir)/mini-howto; done
 
```

Comment:
I'm also pretty sure that it wouldn't be portable, which means opening yet another can of worms (for no good reason).

It's definitely not part of POSIX/SUS.

Reply:
Err, `install` isn't part of POSIX/SUS anyway, so that part was bogus, but I just checked a macOS machine for reference. `BSD install` doesn't know `-t` there, so it must be non-portable `GNU` extension.

Reply:
OK, let's keep it that way